package.json

```diff
@@ -22,14 +22,17 @@
     "@emotion/styled": "^11",
     "@metamask/onboarding": "^1.0.1",
     "@snapshot-labs/snapshot.js": "^0.4.5",
+    "@uiw/react-markdown-editor": "^5.10.1",
```

Comment:
Is this package used somewhere in the form?

Reply:
can we remove unused packages such as `@uiw/react-markdown-editor` and `react-markdown-editor-lite`?